Schema/PublicSchema/ContextTagKeys.bond

```diff
@@ -0,0 +1,114 @@
+
+namespace AI
+
+[ContextContract("Emit")]
+[PseudoType("JSMap")]
+struct ContextTagKeys
+{
+    [Description("Application version. Information in the application context fields is always about the application that is sending the telemetry.")]
+    [MaxStringLength("1024")]
+    10: string 	 ApplicationVersion = "ai.application.ver";
+    
+    [Description("Unique client device id. Computer name in most cases.")]
+    [MaxStringLength("1024")]
+    100: string 	 DeviceId = "ai.device.id";
+    
+    110: string 	 DeviceLanguage = "ai.device.language";
+    
+    [Description("Device locale using <language>-<REGION> pattern, following RFC 5646. Example 'en-US'.")]
+    [MaxStringLength("64")]
+    115: string 	 DeviceLocale = "ai.device.locale";
+    
+    [Description("Model of the device end user of the application using. Used for client scenarios. If empty, derived from user agent.")]
```

Comment:
I'll regenerate. Didn't take the latest changed from DataCollectionSchemas repo. Thanks!